robot/resources/lib/python_keywords/node_management.py

```diff
@@ -169,7 +169,8 @@ def exclude_node_from_network_map(
     storage_node_set_status(node_to_exclude, status="offline")
 
     time.sleep(parse_time(MORPH_BLOCK_TIME))
-    tick_epoch(shell, cluster)
+    current_epoch = get_epoch(shell, cluster)
+    tick_epoch_and_wait(shell, cluster, current_epoch)
```

Comment:
You don't use `current_epoch` here, so why don't you use a function that handles `get_epoch()` internally?

Reply:
> You don't use `current_epoch` here, so why don't you use a function that handles `get_epoch()` internally?

Fixed